packages/http/src/mocker/HttpMocker.ts

```diff
@@ -0,0 +1,64 @@
+import { IMocker, IMockerOpts } from '@stoplight/prism-core/types';
+import { IHttpOperation } from '@stoplight/types';
+import { IHttpConfig, IHttpRequest, IHttpResponse } from '../types';
```

Comment:
Just a small stylistic thing, but lets import all npm modules first, and then a new line, and then all the file imports. I already made this change, and will be pushing it shortly. Just wanted to comment.